renovate.json

```diff
@@ -0,0 +1,113 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "dependencyDashboard": true,
+  "dependencyDashboardOSVVulnerabilitySummary": "all",
+  "osvVulnerabilityAlerts": true,
+  "reviewersFromCodeOwners": true,
+  "extends": [
+    "config:best-practices",
+    "customManagers:dockerfileVersions",
+    "security:openssf-scorecard",
+    "mergeConfidence:all-badges",
+    "github>argoproj/argo-cd//renovate-presets/fix/openssf-merge-confidence-columns.json5",
+    "github>argoproj/argo-cd//renovate-presets/custom-managers/shell.json5",
+    "github>argoproj/argo-cd//renovate-presets/custom-managers/yaml.json5"
+  ],
+  "packageRules": [
+    {
+      "description": "Disable all updates to avoid conflicts with dependabot, then enable what we want",
+      "matchPackageNames": [
+        "*"
+      ],
+      "enabled": false
+    },
+    {
+      "description": "Add label dependencies to all PRs",
+      "matchPackageNames": [
+        "*"
+      ],
+      "labels": [
+        "dependencies"
+      ]
+    },
+    {
+      "description": "Enable node-version",
+      "matchDatasources": [
+        "node-version"
+      ],
+      "addLabels": [
+        "javascript"
+      ],
+      "enabled": true
+    },
+    {
+      "description": "Enable golang-version",
+      "matchDatasources": [
+        "golang-version"
+      ],
+      "addLabels": [
+        "go"
+      ],
+      "enabled": true
+    },
+    {
+      "description": "Enable some go modules",
+      "matchDatasources": [
+        "go"
+      ],
+      "matchPackageNames": [
+        "go",
+        "github.com/golangci/golangci-lint"
+      ],
+      "addLabels": [
+        "go"
+      ],
+      "enabled": true
+    },
+    {
+      "description": "Enable bump of golang version in go.mod",
+      "matchDatasources": [
+        "golang-version"
+      ],
+      "rangeStrategy": "bump"
+    },
```

Comment:
I wonder if we can actually exclude this. iirc the go.mod version is the _minimum_ required version to use our package. It's probably better for us to just manually bump that when people encounter compatibility issues with older versions, rather than locking people out of using older versions.

Reply:
I wasn't sure about that one.
But yes you can remove this rule and renovate will update the golang pr right away.